documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,174 @@ You can add {a-rhv} source provider by using the {ocp} web console.
-. Enter the *SHA-1 fingerprint*.
+.. To skip certificate validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certificates, leave the *Skip certificate validation* check box cleared.
 endif::[]
 ifdef::rhv[]
-. Select *Red Hat Virtualization* from the *Provider type* list.
+. Click *Red Hat Virtualization*
+. Specify the following fields:
+
+* *Provider resource name*: Name of the source provider
+* *URL*: API endpoint of the {rhv-short} Manager on which the source VM is mounted
+* *Username*: Username
+* *Password*: Password
+
+. Choose one of the following options for validating CA certificates:
+
+** *Skip certificate validation* : Migrate without validating a CA certificate.
+** *Use the system CA certificates*: Migrate after validating the system CA certificates.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+
+.. To skip certificate validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certificates, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
+
+endif::[]
+ifdef::ova[]
+. Click *Open Virtual Appliance (OVA)*.
+. Specify the following fields:
+
+* *Provider resource name*: Name of the source provider
+* *URL*: URL of the NFS file share that serves the OVA
+endif::[]
+ifdef::ostack[]
+. Click *OpenStack*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *RHV Manager host name or IP address*: {manager} host name or IP address -  if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *RHV Manager user name*: {manager} user
-* *RHV Manager password*: {manager} password
+* *Provider resource name*: Name of the source provider.
+* *URL*: {osp} Identity (Keystone) endpoint, for example, `http://controller:5000/v3`.
+* *Authentication type*: Choose one of the following methods of authentication and supply the information related to your choice. For example, if you choose *Application credential ID* as the authentication type, the *Application credential ID* and the *Application credential secret* fields become active, and you need to supply the ID and the secret.
+
+** *Application credential ID*
+*** *Application credential ID*: {osp} application credential ID
+*** *Application credential secret*: {osp} https://github.com/kubev2v/forklift-documentation/pull/402pplication credential `Secret`
+
+** *Application credential name*
+*** *Application credential name*: {osp} application credential name
+*** *Application credential secret*: : {osp} application credential `Secret`
+*** *Username*: {osp} username
+*** *Domain*: {osp} domaina error
```

Comment:
```suggestion
*** *Domain*: {osp} domain error
```

Reply:
@anarnold97 Good catch -- "domain error," even properly spelled, makes no sense. Changed to "{osp} domain name" like the others.